_posts/2022-03-07-using-private-image-repositories.markdown

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title:  "Deploying workloads with images from private repositories"
+date:   2022-02-07 17:00:00 +0100
+categories: flotta
+author: Jakub Dżon
+tags:
+  - guide
+  - flotta
+  
+summary-1: Deploying workloads with images from private repositories
+---
+
+Users can deploy their containerized workloads to edge devices using Flotta and very often images they want to use are 
```

Comment:
WDYT do add a sentence to define what a workload/containerized workload is? 
I think It may help to make the context clearer for someone is approaching the topic for the first time.

Reply:
I've added little explanation and a link. I don't think this post is a good place to explain details of containerization - I think it should be described in https://issues.redhat.com/browse/ECOPROJECT-618.